tools/accuracy_checker/accuracy_checker/adapters/pose_estimation.py

```diff
@@ -337,3 +337,66 @@ def get_poses(subset, candidate):
         scores = np.array(scores)
 
         return persons_keypoints_x, persons_keypoints_y, persons_keypoints_v, scores
+
+
+class SingleHumanPoseAdapter(Adapter):
+    __provider__ = 'single_human_pose_estimation'
+    prediction_types = (PoseEstimationPrediction, )
+
+    @classmethod
```

Comment:
you do not need to specify output layer name explicitly if you model has only one output.
adapter has field `output_blob`, which store output layer name as default.
Just remove `parameters`, `validate_config` and `configure` method and change
heatmaps = np.transpose(raw_outputs[self.keypoints_heatmap][-1], (1, 2, 0)) -> np.transpose(raw_outputs[self.output_blob][-1], (1, 2, 0))